FirebaseFunctions.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'FirebaseFunctions'
-  s.version          = '2.0.0'
```

Comment:
Should be 2.1.0. - but actually don't change it.

Update M29 on the release spreadsheet instead and it will be updated automatically by the release script when the next release branch is made. If it gets updated early, it won't integrate with the current Firebase pod.

Reply:
D'ohh! Fixed.